mcu/atmega-hal/src/lib.rs

```diff
@@ -47,6 +52,14 @@ pub struct Peripherals {
     pub USART0: RawPeripheral<pac::USART0>,
     #[cfg(feature = "atmega32u4")]
     pub USART1: RawPeripheral<pac::USART1>,
+    #[cfg(feature = "atmega2560")]
+    pub USART0: RawPeripheral<pac::USART0>,
+    #[cfg(feature = "atmega2560")]
+    pub USART1: RawPeripheral<pac::USART1>,
+    #[cfg(feature = "atmega2560")]
+    pub USART2: RawPeripheral<pac::USART2>,
+    #[cfg(feature = "atmega2560")]
+    pub USART3: RawPeripheral<pac::USART3>,
```

Comment:
I think we can make this a bit more concise by re-using statements that are the same for multiple chips:

```rust
    #[cfg(any(feature = "atmega328p", feature = "atmega2560")]
    pub USART0: RawPeripheral<pac::USART0>,
    #[cfg(any(feature = "atmega32u4", feature = "atmega2560")]
    pub USART1: RawPeripheral<pac::USART1>,
    #[cfg(feature = "atmega2560")]
    pub USART2: RawPeripheral<pac::USART2>,
    #[cfg(feature = "atmega2560")]
    pub USART3: RawPeripheral<pac::USART3>,
```

Reply:
fixed